packages/components/src/components/form-progress/form-progress.jsx

```diff
@@ -3,82 +3,73 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Text from '../text';
 
-class FormProgress extends React.PureComponent {
-    constructor(props) {
-        super(props);
-        this.el_completed_bar = React.createRef();
-    }
+const FormProgress = ({ steps, current_step }) => {
+    React.useEffect(() => {
+        animateCompleteBar();
+    });
```

Comment:
Do you want this `useEffect` to run on every render or just when the component is mounted?